.github/workflows/update-packages.yaml

```diff
@@ -0,0 +1,42 @@
+name: Update packages of Jupyter images
+
+on: 
+  workflow_dispatch:
+    inputs:
+      package:
+        description: 'Package Name'     
+        required: true
+        default: ''    
+      version:
+        description: 'Package Version'     
+        required: true
+        default: ''
+     
+jobs:
+  update-package:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        ref: main
+
+    - name: Update package version
+      run: |
+        echo "Updating information for package = ${{ github.event.inputs.package }} version = ${{ github.event.inputs.version }}"
+        export PKG_NAME_LOWER=$(echo " ${{ github.event.inputs.package }}" | tr '[:upper:]' '[:lower:]')
+        export PKG_VER=$(echo ${{ github.event.inputs.version }} | sed -e "s/v//g")
+
+        DIRS=("base" "swan" "swan-cern")
+        for DIR in "${DIRS[@]}"; do
+          sed -i "s/\( $PKG_NAME_LOWER==*\)[^ ]*/\1$PKG_VER/" "$DIR/Dockerfile"
```

Comment:
Shall we also consider the case where multiple packages are in one line? (Don't think this was the case in the past...)

Reply:
Actually, are you sure this works? If I have `\` in the same line, would it match?